### PR TITLE
HT-3038: Individual etas reports

### PR DESCRIPTION
**Pull request by jsteverman**
The previous version only produced records for holdings that matched HT Items. This version has at least one record for every holding.

### CODE REVIEW

**Review by aelkiss — APPROVED** [review]
Looks good to me; definitely glad to see specs added here. If performance is still lacking, it would probably be worth doing some profiling to verify where the issues are. I'm not seeing any obvious problems where it's iterating over the cluster multiple times unnecessarily. I do wonder how well `missed_holdings` will perform,  but at least we're only calling it once per cluster.

File: bin/export_etas_overlap_report.rb [hunk 1]
```diff
@@ -13,41 +13,83 @@
...
+
+  def missed_holdings(cluster, holdings_matched)
+    if organization.nil?
+      cluster.holdings - holdings_matched.to_a
+    else
+      cluster.holdings.group_by(&:organization)[organization] - holdings_matched.to_a
+    end
+  end
+
+  def write_overlaps(cluster, organization)
```
Comment (mwarin) [inline bin/export_etas_overlap_report.rb:64]: I'd like to see the different outputs of write_overlap (what is passed to write_record and what is returned) explained in a comment.

**Review by mwarin — COMMENTED** [review]
I don't think bin/export_etas_overlap_report.rb needs to require zinzout or waypoint anymore.

**Review by mwarin — APPROVED** [review]
OK with me.

**Review by mwarin — APPROVED** [review]

**Pull request merged**